apps/publishers/services/publisher_member_notification_service.py

```
@@ -0,0 +1,63 @@
+    def __init__(
+        self,
+        repo: PublisherMemberInvitationRepository | None = None,
+        member_repo: PublisherMemberRepository | None = None,
+    ) -> None:
+        self.repo = repo or PublisherMemberInvitationRepository()
+        self.member_repo = member_repo or PublisherMemberRepository()
+
+    def send_invitation_email(self, invitation_id: int, raw_token: str) -> None:
+        try:
+            inv = self.repo.get_for_email(invitation_id)
+        except Exception:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)